app/javascript/common/helpers.js

```diff
@@ -1,14 +1,16 @@
 // todo: expose this from patternfly-react instead?
+export const getDisplayName = Component =>
+  Component.displayName || Component.name || 'Component';
+
+// TODO upgrade to latest patternfly-react after next release, these are exposed now:
```

Comment:
ok yep - we can do this in a subsequent PR...

Reply:
ok fine... let's go ahead and fix this!

Reply:
just update pf-react...and it should be there...

Reply:
Done.